docs/faq.md

```diff
@@ -2,53 +2,113 @@
+  - You can put bounties (money) on issues or suggestions that you want fixed using bountysource. Learn more here: [Bountysource](bountysource.md)
+  - You can help improve the wiki by submitting a [Pull Request](https://github.com/azerothcore/wiki).
 
-Read this carefully: [How to ask for help](how-to-ask-for-help.md).
+- Why do you not merge my fix?
+  - All fixes must be reviewed by a dev. All devs don't know everything, so you need to wait until someone reviews it.
+  - Some fixes requires testing and not all devs can test so you need to wait on someone else to test it.
+  - They get merged eventually after getting the To Be Merged label.
+  - Maybe you are not following the [SQL/C++ standards](https://www.azerothcore.org/wiki/standard-operating-procedure).
+  - Read more in [Merge Process](merge-process.md).
 
-#### 5 - Why is it better to ask on Stackoverflow for support questions?
+- How do I report a crash?
+  - By pasting your crashlog into a PasteBin or Gist.
+  - The crashlog **needs to come from a RelWithDebInfo or Debug compilation**. If it is from Release it is useless.
 
-- Because your question (and its solutions) **stays there** and you or other users **can find it later**, even using **google**
-- Because your question can easily **get lost in the chat**, and asking in the chat can only get answers by the users who are **currently online**
-- Because it's **super easy** (you can log in Stackoverflow using your **Google** or **Facebook** account - **NO NEED TO CREATE A NEW ACCOUNT**)
-- Because you can add common **tags** to your questions (like `c++`, `sql`, `docker`, `lua`, etc...) and have the chance to  **receive help from users** that you would not find in the AC discord chat
-- Because it gives extra **visibility** to our project
-- Because it helps to **avoid duplicate questions**
-- Because by posting questions or answers you can earn **points** (a profile with many points Stackoverflow can help A LOT when you want to find a job as a developer)
+- Do you support Repacks based on AzerothCore?
+  - No. Compile the code yourself if you want support.
 
-#### 6 - Why does AzerothCore not mimic blizzlike bugs or exploits?
+## Database-related FAQ
 
-Although we strive to provide blizzlike content we also value user experience. This means that we sometimes fix bugs or exploits which existed during retail at the time to provide a better overall experience for the players.
+- How often do you update the database?
+  - The database is updated nearly every day.
 
+- How do I update the database?
+  - You can find everything you need to keep the database up-to-date in this guide on [Database Keeping the Server Up-to-Date](database-keeping-the-server-up-to-date.md).
 
+## Core-related FAQ
 
-## COMMON ERRORS
+- When will the source be stable?
+  - Soon...™
```

Comment:
```suggestion
  - We do our best to keep the `master` branch stable and playable. We never push code directly into the `master` branch, but rather we first require everyone (including AC admins and staff) to first open a PR so everyone is able check them before they got merged into `master`. Please help us by [testing PRs](https://www.azerothcore.org/wiki/How-to-test-a-PR) and reporting any bugs you may find.
 - We recommend updating your core frequently, at least once a week, in order to benefit of latest core fixes and security patches.
```

Reply:
Fine, but I am not removing Soon...TM, too much of a classic to be forgotten :D